packages/styles/dropdown.css

```diff
@@ -1,44 +1,3 @@
-.Dropdown {
```

Comment:
As far as I can tell, these styles weren't documented anywhere, but is there any concern about this being a breaking change if any code was using `.Dropdown`?

Reply:
I'm not 100% sure why those styles were even there. They probably should've exclusively been a part of the OptionsMenu styles, but I'm leaving them there for now.